src/signTx.c

```diff
@@ -383,14 +397,35 @@ static void signTx_handleInitAPDU(uint8_t p2, uint8_t* wireDataBuffer, size_t wi
 			// in WITNESS stage we do not verify whether the witness belongs
 			// to a given utxo, withdrawal or certificate.
 
-			size_t maxNumWitnesses;
-			if (ctx->commonTxData.isSigningPoolRegistrationAsOwner) {
+			size_t maxNumWitnesses = 0;
+			switch (ctx->commonTxData.signTxUsecase) {
+			case SIGN_TX_USECASE_POOL_REGISTRATION_OWNER:
 				maxNumWitnesses = 1;
-			} else {
+				break;
+
+				#ifdef POOL_OPERATOR_APP
+			case SIGN_TX_USECASE_POOL_REGISTRATION_OPERATOR:
+				ASSERT(ctx->numCertificates == 1);
+				// inputs and withdrawals are unrestricted, to fund the tx
+				// only a single pool registration certificate
+				// with two possible witnesses: pool key and one owner
+				maxNumWitnesses = (size_t) ctx->numInputs +
```

Comment:
imho there's not much point in allowing withdrawals for pool registration from operators' perspective if the owners would not be allowed to sign the transaction - technically, they can  use the cli, but I can see that inconsistency introducing significant confusion as to why operator's tx went through and owner's didn't

Reply:
if operators want to withdraw rewards from their pool, they can do so simply by creating a separate withdrawal transaction without any certificates